Readme.md

```diff
@@ -0,0 +1,24 @@
+# How to use the split utility
+
+The split utility is used by giving it a LLVM bitcode file and it splits all of
+the functions in it into separate files. 
+
+Obtaining the final LLVM bitcode file can be dune in multiple ways. The easiest
+one being to use
+[whole-program-llvm](https://github.com/travitch/whole-program-llvm) that gives
+you the bitcode used to create the executable.
+
+For example to get the bitcode from the cpython executable you build it like
+that: 
+```
+LLVM_COMPILER="clang" CC="wllvm" ./configure 
+LLVM_COMPILER="clang" make
+``` 
+Then extract the bitcode file from the binary with:
+```
+extract-bc python 
```

Comment:
Where does `extract-bc` come from?

Reply:
This is part of whole-program-llvm

Reply:
Let's say something like "Using the `extract-bc` program from whole-program-llvm, extract the bitecode file from the binary as follows:".

Reply:
Changed in: a16afc8c604a3a01bf624b5e90550e02dd126d30